src/components/Organisms/Header/Nav/Nav.js

```diff
@@ -85,7 +86,9 @@ const MainNav = ({ navItems }) => {
 
             /* Determine which field represents our url path */
             let thisUrl = NavHelper(thisFirstChild);
-
+            const relNoopener = !whiteListed(thisUrl) && 'noopener';
+            const hasSubmenu =
+              group.links && group.links.length > 1 ? 'true' : null;
```

Comment:
Can we not use true boolean values here? 

A `"true"` string and `null` seem a bit weird..

Reply:
Just checked. I can't use it for submenu logic. I will change it 👍 

I need this way for `haspopup` to ignore the attribute if null but keep it and add string "true".

Reply:
_Personally_, I'd keep that `hasSubMenu` boolean logic up top (nicer to see the vars together beforehand), and just made a second var for `hasPopUp`, based on `hasSubMenu`:

`hasPopUp = hasSubMenu ? 'true' : null
`